docs/03.API-EXAMPLE.md

```diff
@@ -551,6 +551,32 @@ Value of x is 12
 Value of x is 17
 ```
 
+## Changing the seed of pseudorandom generated numbers
+
+If you want to change the seed of Math.random() generated numbers, you have to initialize srand with a seed value.
+A recommended method is using `jerry_port_get_current_time(void)` or something based on a constantly changing value, therefore every run produces truly random numbers.
+
+[doctest]: # ()
+
+```c
+#include <string.h>
+#include <stdlib.h>
+#include "jerryscript.h"
+#include "jerryscript-port.h"
+
+int
+main (void)
+{
+  const jerry_char_t script[] = "var a = Math.random ();";
+  size_t script_size = strlen ((const char *) script);
+
+  srand ((unsigned) (jerry_port_get_current_time ()));
```

Comment:
`(unsigned) jerry_port_get_current_time ()` should be enough (no need for the extra parentheses)

also in all the "real" `main` functions